script/Ciel/Day 4/04_01_CIEL04_1.txt

```diff
@@ -177,87 +192,101 @@ It's not like I hate you or anything."
+I overstepped my bounds, didn't I...
+ Plus, serving convenience store food for dinner doesn't reflect well on me as a girl..."
 }
 [sha:2d7691391d38d406c865b0fdc0268984107cf52b]{
 -- Page 18, Offset 8232.
 -- 先輩は悲しそうに目をふせた。
-Senpai sadly closed her eyes.
+Senpai closes her eyes, crestfallen.
 }
 [sha:73b349eaa8b343e1beab5152c11c143dc00ded9c]{
 -- Page 18, Offset 8233.
 -- 俺にとって先輩は昨夜から特別な―――なんていうか、人間として尊敬するにあまりある―――人になっている。
-Since last night, she has become a very special person to me, someone I have a lot of respect for as a human being.
+After last night, she's become someone very special to me―――someone I have a lot of respect for as a human being.
 }
 [sha:8f1bc806d722a5a300f017280b177c0f7423002c]{
 -- Page 18, Offset 8234.
 -- そんな人に悲しげにされるとこっちが参ってしまう。
-When someone like that is sad, I feel hurt as well.
+I can't bear to see a person like that feeling sad.
 }
 [sha:d0849b43ffe6862a62058142b7d34abcf01af921]{
 -- Page 19, Offset 8235.
 -- 「本気で違います、お節介なんかじゃないです！
-"No, seriously, you're not meddling!
+"No, seriously, you didn't overstep anything!
 }
 [sha:4d4864542408202368d474251ddde4760bdaf255]{
 -- Page 19, Offset 8236.
 -- あの……俺は、本当に感謝しています。先輩が介抱してくれなかったら俺はまた間違えるところだったし、コンビニ弁当もアリだし、べ、ベッドは緊張したけどすぐ眠れたし！
-You see... I'm really grateful. If it wasn't for you taking care of me, I would have made another mistake, and convenience store lunches are alright and I was nervous about using your bed, but I fell asleep right away!
+Um...
+ I really am grateful.
+ I would have ended up making an even bigger mistake if you hadn't taken care of me,
+ I think convenience store meals are totally fine,
+ a-and I was really nervous about sleeping in your bed, but I ended up falling asleep right away!
 }
 [sha:fca2292ca0435c180038601ac2081090fd1ddae5]{
 -- Page 19, Offset 8237.
 -- 今だってその、先輩の顔が見れて嬉しいし、気持ちがめちゃくちゃ楽になってるんですから！」
-Even now, I'm happy to see your face, and it makes me feel so much better!"
+Right now, you know,
+ I'm just happy to see your face,
+ it really puts me at ease, so...!"
 }
 [sha:2b47ac157c5b516103b0420eae765b57f39472e6]{
 -- Page 20, Offset 8238.
 -- とにかく思いついた感謝をまくしたてる。
-I just rattled off all the thanks I could think of.
+I just rattle off all the thanks I can think of.
 }
 [offset:8239]{
 -- Page 20, Offset 8239.
 -- ……と。
-――――――――――
+And then―――
 }
 [sha:d306710c4ea62c455445cd0b6d5a66d64a01db13]{
 -- Page 21, Offset 8240.
 -- 「はい、そうですね。遠野くんはいつもの調子に戻ってくれました。昨夜は何があったのかは知りませんけど、こうして元気になってくれて嬉しいです」
-"I see. You're back to normal. I don't know what happened last night, but I'm glad you're feeling better."
+"Ah, there we go.
+ Seems like you're all back to normal.
+ I don't know what happened last night, but I'm glad you're feeling better."
 }
 [sha:ddcac14bb15624eff7ce3fc16ead5f14ffe8f953]{
 -- Page 22, Offset 8241.
 -- さっきの悲しそうな顔はどこにいったのか、先輩はくすくすと笑っている。
-While I wonder where her sad face from earlier went, Senpai chuckles.
+Senpai chuckles softly while I try and work out where the sad face from earlier disappeared to.
```

Comment:
from earlier went.

Reply:
I feel like disappeared to is more idiomatic? Maybe it's en_GB

Reply:
sounds pretty wrong to me so i'd say it's probably an atlantic divide